.github/workflows/windows.yml

```diff
@@ -15,7 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-name: Apache Netbeans
+name: Windows
```

Comment:
nitpick: i would keep the old name here until the other jobs are updated or update the "Check Linux" Job too for consistency.

Reply:
@mbien I've updated the other workflow names and the badges in the readme.

Reply:
@oyarzun excellent